spec/features/delete_genotype_spec.rb

```diff
@@ -2,8 +2,8 @@
   let(:user) { create(:user) }
   let!(:genotype) { create(:genotype, user: user) }
   let!(:user_achievement) do
-    create(:user_achievement, achievement: Achievement.find_by(award: 'Published genotyping'),
-                              user: user)
+    create(:user_achievement, achievement_id: Achievement.find_by(award: 'Published genotyping').id,
+                              user_id: user.id)
```

Comment:
Doesn't the test rather fail due to the missing seeds? Whether you use the instance or the id shouldn't make any difference, except that it looks nicer using the instances. ;)